src/doc/errors.ts

```diff
@@ -1487,6 +1516,28 @@ export const doc_errors:Documentation = {
 			Array.sort. Given two items as arguments, this function must
 			return a numeric value. If the value is non-numeric, then this
 			error is reported.
+            <div class="example">
+            <h3>Example</h3>
+            <tscript>
+var arr = [1, 4, 2, 3];
+
+function order1(a, b) {
+    if (a<=b) then return -1; 
```

Comment:
Remove parentheses, otherwise this is a C-style if statement. Same comment refers to the similar if-statements below.